packages/concerto-core/lib/introspect/modelfile.js

```diff
@@ -389,6 +389,18 @@ class ModelFile extends Decorated {
         }),this);
     }
 
+    /**
+     * Returns the actual imported name from another namespace
+     * @param {string} type - the short name of the type
+     * @returns {string} - the actual imported name. If not aliased then returns the same string
+     */
+    getActualImportType(type) {
```

Comment:
Can you add a unit test for this function please?

Reply:
added the unit test-cases

Reply:
let's call this,`getImportedType`

Reply:
Sure. 👍🏻